sim/scenarios/drop-rate/drop-rate-error-model.cc

```diff
@@ -13,12 +13,15 @@ TypeId DropRateErrorModel::GetTypeId(void) {
 }
  
 DropRateErrorModel::DropRateErrorModel()
-    : rate(0), rng(rd()), distr(0, 99) { }
+    : rate(0), distr(0, 99) {
+    std::random_device rd;
+    rng = new std::mt19937(rd());
+}
```

Comment:
Just for future reference, there was an argument about how to properly seed the random number generator here: https://codereview.stackexchange.com/questions/109260/seed-stdmt19937-from-stdrandom-device.

I think what we're doing here is perfectly fine for our purposes.